compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2039,6 +1999,67 @@ static LogicalResult adjustTileSizesForUnPackOp(func::FuncOp entryPointFn,
                                                tileSizesList, pipeline);
 }
 
+static void setLoweringConfigForComputeOps(func::FuncOp entryPointFn,
```

Comment:
Maybe some comments about what is happening here?

Reply:
+1

Reply:
done, please take a look

Reply:
Thanks! That's very helpful